python/etddf/sim.py

```diff
@@ -159,20 +161,99 @@ def init_agents(self,x_true_vec,dynamics_fxn='lin_ncv',sensors={}):
 
         for i in range(0,self.num_agents):
 
-            agent_id = i
+            # agent_id = i
 
-            ids = deepcopy(self.connections[i])
+            # ids = deepcopy(self.connections[i])
```

Comment:
Does this block comment need to be here? It looks like all of the 4-states-per-agent code was copied, commented out, and updated to 6-states-per below.